config

```diff
@@ -30,6 +30,13 @@ ECHO_DEPS="                                                                 \
         $ngx_addon_dir/src/ngx_http_echo_foreach.h                          \
         "
 
+# nginx won't have HTTP_POSTPONE_FILTER_MODULE & HTTP_POSTPONE_FILTER_SRCS
+# defined since 1.9.11
+if test -z "$HTTP_POSTPONE_FILTER_MODULE"; then
+    HTTP_POSTPONE_FILTER_MODULE=ngx_http_postpone_filter_module
+    HTTP_POSTPONE_FILTER_SRCS=src/http/ngx_http_postpone_filter_module.c
+fi
```

Comment:
Just to chime in on this with some confirmation. Skipping those will break building this `--without-http_ssi_module` or something else to pull in HTTP_POSTPONE.

I feel like what we really need is a better way to tell Nginx we depend on that and it needs to be included.